frontend/pages/email.js

```diff
@@ -1,8 +1,52 @@
-import React from "react";
+import React, { useState } from "react";
 import Link from "next/link";
-
+import axios from "axios";
+import { ToastContainer, toast } from "react-toastify";
+import 'react-toastify/dist/ReactToastify.css';
 
 const Email = () => {
+
+const [email, setEmail] = useState("");
+
+function displayMessage (status, message) {
+
+    if (status === 201) {
+    toast.success(
+        message,{
+            position: toast.POSITION.TOP_RIGHT
+     });
+    } else {
+     toast.error(
+        message, {
+            position: toast.POSITION.TOP_RIGHT
+    }); 
+    }
+}
+
+function onChangeEmail (evt) {
+    setEmail(evt.target.value);
+}
+
+async function getEmailFromForm (evt) {
```

Comment:
Recomendo alterar o nome da função **getEmailFromFrom** para algo como **registerEmailInNewsletter**. OBS: lembre-se que o nome de um função é para explicar o que ela faz de forma resumida e também isso ajudar a pessoa que for mexer nesse código possa entender o que o código faz apenas lendo o nome da função, ok?